src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -1187,6 +1275,254 @@ ResolutionComplete:
             Return New OverloadResolutionResult(candidates.ToImmutable(), resolutionIsLateBound, narrowingCandidatesRemainInTheSet, asyncLambdaSubToFunctionMismatch)
         End Function
 
+        Private Shared ReadOnly s_poolInstance As ObjectPool(Of PooledDictionary(Of NamedTypeSymbol, NamedTypeSymbol)) =
```

Comment:
>s_poolInstance [](http://example.com/codeflow?start=32&length=14)

Is this used?

Reply:
> Is this used?

Thanks for catching this. It is meant to be used for ```candidateInfoByDeclaringType``` in the method below.

Reply:
This wouldn't affect the result though since we are comparing definitions

Reply:
Addressed in #75941